Quarkus Kafka Streams extension does not start in a DEV mode

### Describe the bug

I use Quarkus Kafka Streams extension and cannot use DEV mode. I can only reproduce this with 999-SNAPSHOT.

### Expected behavior

I can use DEV mode.

### Actual behavior

Startup fails and exception is thrown:

```
mvavrik@fedora:/tmp/repro$ quarkus dev -Dquarkus.platform.version=999-SNAPSHOT -Dquarkus.platform.group-id=io.quarkus
[INFO] Scanning for projects...
[INFO] 
[INFO] ---------------------------< org.acme:repro >---------------------------
[INFO] Building repro 1.0.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- quarkus:999-SNAPSHOT:dev (default-cli) @ repro ---
[INFO] Invoking resources:3.3.1:resources (default-resources) @ repro
[INFO] Copying 1 resource from src/main/resources to target/classes
[INFO] Invoking quarkus:999-SNAPSHOT:generate-code (default) @ repro
[INFO] Invoking compiler:3.14.0:compile (default-compile) @ repro
[INFO] Nothing to compile - all classes are up to date.
[INFO] Invoking resources:3.3.1:testResources (default-testResources) @ repro
[INFO] skip non existing resourceDirectory /tmp/repro/src/test/resources
[INFO] Invoking quarkus:999-SNAPSHOT:generate-code-tests (default) @ repro
[INFO] Invoking compiler:3.14.0:testCompile (default-testCompile) @ repro
[INFO] No sources to compile
Listening for transport dt_socket at address: 5005
Exception in thread "main" java.lang.RuntimeException: java.util.ServiceConfigurationError: io.quarkus.dev.spi.HotReplacementSetup: Provider io.quarkus.kafka.streams.runtime.devmode.KafkaStreamsHotReplacementSetup not found
	at io.quarkus.deployment.dev.DevModeMain.start(DevModeMain.java:113)
	at io.quarkus.deployment.dev.DevModeMain.main(DevModeMain.java:70)
Caused by: java.util.ServiceConfigurationError: io.quarkus.dev.spi.HotReplacementSetup: Provider io.quarkus.kafka.streams.runtime.devmode.KafkaStreamsHotReplacementSetup not found
	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:593)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.nextProviderClass(ServiceLoader.java:1219)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1228)
	at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1273)
	at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1309)
	at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1393)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.setupRuntimeCompilation(IsolatedDevModeMain.java:267)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:425)
	at io.quarkus.deployment.dev.IsolatedDevModeMain.accept(IsolatedDevModeMain.java:55)
	at io.quarkus.bootstrap.app.CuratedApplication.runInCl(CuratedApplication.java:143)
	at io.quarkus.bootstrap.app.CuratedApplication.runInAugmentClassLoader(CuratedApplication.java:98)
	at io.quarkus.deployment.dev.DevModeMain.start(DevModeMain.java:107)
	... 1 more
```

### How to Reproduce?

Reproducer:

```
quarkus create app repro -x=kafka-streams
cd repro
quarkus dev -Dquarkus.platform.version=999-SNAPSHOT -Dquarkus.platform.group-id=io.quarkus
```

### Output of `uname -a` or `ver`

Fedora, Ubuntu

### Output of `java -version`

OpenJDK Runtime Environment Temurin-21.0.6+7

### Quarkus version or git rev

999-SNAPSHOT (HEAD on 201a31cedad)

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Maven 3.9.9

### Additional information

_No response_

--- a/extensions/kafka-streams/runtime/src/main/resources/META-INF/services/io.quarkus.dev.spi.HotReplacementSetup
+++ b/extensions/kafka-streams/runtime/src/main/resources/META-INF/services/io.quarkus.dev.spi.HotReplacementSetup
@@ -1,1 +1,1 @@
-io.quarkus.kafka.streams.runtime.devmode.KafkaStreamsHotReplacementSetup
+io.quarkus.kafka.streams.runtime.dev.KafkaStreamsHotReplacementSetup

